src/views/ProxyPage.vue

```diff
@@ -421,6 +431,10 @@ export default {
         padding: 0;
     }
 
+    #print-content.with-cut-lines img {
+        margin: 0 1px 1px 0;
```

Comment:
This is a pretty clever way to handle it. I like it.
I had been looking at drawing borders and backgrounds and it was creating all sorts of issues with when things would cross page boundaries.